.github/workflows/test.yml

```diff
@@ -206,7 +206,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        queryEngine: ['library', 'binary']
+        queryEngine: ['binary']
```

Comment:
Isn't the removal of `library` here a bit scary?

Reply:
the test only fails on `binary`, `library` is temporarily removed to make iterating on code and debugging easier